MEETINGS.md

```diff
@@ -2,12 +2,13 @@
 Please find meetings of the GitOps Working Group available below. The next meeting will be listed at the top of the list with links to recordings of past meetings available below (when possible). Times listed are GMT.
 
 ## Upcoming Meetings
-Thursday, February 11th at 18:00 GMT
-[Video Conference Link](https://weaveworks.zoom.us/j/93181235360)<br>
-Meeting ID: 93181235360
-<br><br><br>
+Proposed next meeting: We need to pick regular monthly cadence to be on CNCF calendar -2nd thursday 6pm GMT
 
 ## Past Meetings
+[Video Recording](https://weaveworks.zoom.us/rec/share/OPy5MSMzgIkPX1-PAdhwrnGGa82Lr5BuVpQ1iSK-sbHPph68ML1JNVQXRKlP09bw.bjqDnPKEA2W9D2HG)<br>
```

Comment:
Should the date/time be added as per the other recordings?

i.e. Thursday, February 11th at 18:00 GMT